### DISCUSSION TITLE
Animation utility of tailwind is not working

### DISCUSSION THREAD

**Discussion by ajmeraaxesh** [tailwindlabs/tailwindcss#2074] [Help]
I have the following  defined in tailwind.config.js and I am using tailwind v1.5.2 Am I missing something. Also It seems that tailwind is not gneration the animation variants in the css. Would appreciate any help

module.exports = {
    theme: {
        extend: {
            colors: {
                'brand-blue': '#369dd6',
           
            },
          
        },
        animation: {
            none: 'none',
            spin: 'spin 1s linear infinite',
            ping: 'ping 1s cubic-bezier(0, 0, 0.2, 1) infinite',
            pulse: 'pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite',
            bounce: 'bounce 1s infinite',
        },
        keyframes: {
            spin: {
                from: { transform: 'rotate(0deg)' },
                to: { transform: 'rotate(360deg)' },
            },
            ping: {
                '0%': { transform: 'scale(1)', opacity: '1' },
                '75%, 100%': { transform: 'scale(2)', opacity: '0' },
            },
            pulse: {
                '0%, 100%': { opacity: '1' },
                '50%': { opacity: '.5' },
            },
            bounce: {
                '0%, 100%': {
                    transform: 'translateY(-25%)',
                    animationTimingFunction: 'cubic-bezier(0.8,0,1,1)',
                },
                '50%': {
                    transform: 'translateY(0)',
                    animationTimingFunction: 'cubic-bezier(0,0,0.2,1)',
                },
            },
        },
    },
    variants: {
        animation: ['responsive']
    },
    purge: [
        './src/**/*.html',
        './src/**/*.vue',
        './src/**/*.jsx',
        './src/**/*.js',
        './public/**/*.js',
        './public/**/*.html',
    ],
}

**Comment by gsimone** [comment]
It's not yet included, the PR is still open
